app/src/lib/stores/app-store.ts

```diff
@@ -316,8 +316,10 @@ const externalEditorKey: string = 'externalEditor'
 const imageDiffTypeDefault = ImageDiffType.TwoUp
 const imageDiffTypeKey = 'image-diff-type'
 
-const hideWhitespaceInDiffDefault = false
-const hideWhitespaceInDiffKey = 'hide-whitespace-in-diff'
+const hideWhitespaceInChangesDiffDefault = false
+const hideWhitespaceInChangesDiffKey = 'hide-whitespace-in-changes-diff'
+const hideWhitespaceInHistoryDiffDefault = false
+const hideWhitespaceInHistoryDiffKey = 'hide-whitespace-in-diff'
```

Comment:
```suggestion
const hideWhitespaceInHistoryDiffKey = 'hide-whitespace-in-history-diff'
```

I did want to do something like this but, then you'd loose your setting value when upgrading

Reply:
Makes sense